feature-configs/cn-ran-overlays/all-in-one/site.1.fqdn/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Is the addition of the network here worth the complexity of having an extra layer?
This is tied 1:1 with the network policy and with the namespace the workload will run into. 
Would it make sense to have the network too as part of the example layer?

Reply:
This is because of the assumption that there will be many sites using the same hardware (same NIC port names in the SriovNetworkNodePolicy, example folder ), but different VLANs and / or IP settings (site folders).